static/app/components/stream/groupChart.tsx

```diff
@@ -19,70 +19,49 @@ function asChartPoint(point: [number, number]): {name: number | string; value: n
 const EMPTY_STATS: ReadonlyArray<TimeseriesValue> = [];
 
 type Props = {
-  data: Group;
-  statsPeriod: string;
+  stats: ReadonlyArray<TimeseriesValue>;
   height?: number;
+  secondaryStats?: ReadonlyArray<TimeseriesValue>;
   showMarkLine?: boolean;
   showSecondaryPoints?: boolean;
```

Comment:
The `statsPeriod` props was only being used to extract the correct series from the `group` prop. The GroupChart component shouldn't care about the group itself or which time period is being displayed, it should only care about and receive the actual series data itself, hence why the `stats` and `secondaryStats` props replace this. 

This delegates the logic of extracting the series from the group itself to the parent component of groupChart